packages/devtools_app/integration_test/test_infra/run/run_test.dart

```diff
@@ -290,8 +289,10 @@ class TestRunnerArgs {
 
     final argWithTestAppDevice =
         args.firstWhereOrNull((arg) => arg.startsWith(testAppDeviceArg));
-    testAppDevice = argWithTestAppDevice?.substring(testAppDeviceArg.length) ??
-        'flutter-tester';
+    testAppDevice = TestAppDevice.fromArgName(
+      argWithTestAppDevice?.substring(testAppDeviceArg.length) ??
+          'flutter-tester',
```

Comment:
nit: use TestAppDevice.flutterTester.argName instead of the 'flutter-tester' string